tests/test_check_files_checksums.py

```diff
@@ -362,7 +367,6 @@ def should_mark_asset_as_replaced_in_new_version() -> None:
             multihash=storage_asset_multihash,
             exists_in_staging=True,
         ):
-
```

Comment:
Line removal probably doesn't belong in this PR.

Reply:
Reverted. Thanks.